core/src/scss/components/card/_card--icon.scss

```diff
@@ -0,0 +1,62 @@
+@charset "UTF-8";
+
+.su-card--icon {
+  &,
+  &-font {
+    @include modular-spacing('padding', 2 null null);
+    text-align: center;
+
+    .su-card__contents {
+      @include modular-spacing('padding', 0 null null);
+    }
+
+    &.su-card--minimal {
```

Comment:
I see you are allowing for multiple variants on this card. With our current variant approach, we do not allow for this as these combinators create exponential complexity. I have found myself wanting these combinators in work that I have done as well but it will be a change to our approach. As we are changing to the v6 branch soon and we have a palooza sprint we should take this chance to talk about the variant vs decorator models.

Reply:
Yeah, I did write the code to allow for combinations of e.g. su-card-minimal and su-card-horizontal with the media variants. I can see how this might make things harder to maintain. Let's talk more about this next week.